query/src/test/java/org/infinispan/query/blackbox/ClusteredQueryTest.java

```diff
@@ -446,6 +449,28 @@ public void testBroadcastSortedInfinispanQuery() {
       StaticTestingErrorHandler.assertAllGood(cacheAMachine1, cacheAMachine2);
    }
 
+   @Test
```

Comment:
The ```@Test``` annotation is not needed.

Reply:
All the other tests are using it, just wanted to follow the pattern

Reply:
not in this test class